src/components/web3/ExceptionListPanel.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { ReactNode, useState, Dispatch } from "react";
+import { bodyText } from "./styles";
+
+interface Props {
+  header: string;
+  subhead: string;
+  web3Address: string;
+  web3Account: "ETH" | "SOL" | null;
+  isExceptionAddressWrong?: boolean;
+  setIsExceptionAddressWrong?: (val: boolean) => void;
+  children?: ReactNode;
+  exceptionList: string[];
+  compareList: string[];
+  onChange: Dispatch<string[]>;
+}
+
+export const ExceptionListPanel: React.FC<Props> = ({
+  header,
+  subhead,
+  web3Account,
+  web3Address,
+  isExceptionAddressWrong,
+  setIsExceptionAddressWrong,
+  exceptionList,
+  compareList,
+  onChange,
+}) => {
+  const [inputText, setInputText] = useState("");
+  const [invalidAddresses, setInvalidAddresses] = useState<string[]>([]);
+  const [validAddresses, setValidAddresses] = useState<string[]>([]);
+
+  const placeholderText =
+    web3Account === "ETH"
+      ? "0x8d247168113b8aa0f020b2441e98b613b9878qmc, ..."
+      : "BtsXBPrf69qEWQrrxVENGvZY2tqFUgZB9zbrpa4t4Rfo, ...";
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setInputText(event.target.value);
+  };
+
+  const validateAddresses = () => {
+    const delimitersRegex = /,|\s|\t|\n/; // Regex to match comma, space, tab, or newline
+    const addresses = inputText
+      .split(delimitersRegex)
+      .map((address) => address.trim());
```

Comment:
My general rule is that any regex should have a unit test to validate.

Regex's pro and their con is that they are very compact. This means they are very difficult to validate by inspection, and therefore difficult to maintain going forward.

Suggest you add a separate function like:

```typescript
export function splitAddresses(addr: string): string[] {
    return inputText
      .split(/,|\s|\t|\n/)
      .map((address) => address.trim());
}
```

Which would then enable you to write a simple jest test that validates the behaviour. Meaning that we have confidence that this regex is correct, and more importantly have the ability to safely modify it in the future when requirements change.